pipelines/run-generic-pipelines-on-apache-airflow/README.md

```diff
@@ -0,0 +1,275 @@
+
+   The DAG name is derived from the pipeline name and concatenated with the current timestamp. Therefore the GitHub repository contains a DAG file for each pipeline run that you initiate from the Visual Pipeline Editor or the Elyra CLI.
+
+   ![Pipeline run submitted confirmation message](doc/images/run-submission-confirmation.png)
+
+   The confirmation message contains three links:
+    - _GitHub repository_: Opens the GitHub repository location where the DAG was saved.
+
+      ![DAG in GitHub repo](doc/images/dag-in-github-repo.png)
+
+      > If this link returns a 404 error, make sure you are logged in to GitHub and your id is authorized to access the repository.
+
+    - _Run details_: Links to the Apache Airflow GUI where you monitor the pipeline execution progress.
+    - _Object storage_: Links to cloud storage bucket where the input artifacts and output artifacts are stored.
+
+1. Open the run details and object storage links in a new browser tab or window.
+
+### Monitor the DAG run
+
+Elyra does not provide a monitoring interface for Apache Airflow. However, it does provide a link to the Apache Airflow GUI.
+
+1. Open the _Run Details_ link to access the Apache Airflow GUI. 
+
+   ![Inspect DAG](doc/images/inspect-dag-status.png)
+
+   The generated DAG is configured to run only once. When the DAG is executed depends on how frequently Apache Airflow polls the GitHub repository for changes. 
+
+1. Click on the `hello-generic-world` DAG and select the _Graph View_ to access the task information.
+
+   ![Inspect DAG tasks](doc/images/inspect-dag-tasks.png)
+
+1. Click on any completed task and open the log file.
+
+   ![Review task log](doc/images/review-task-log.png)
+
+   Note that the task is implemented using the [`NotebookOp` operator](https://github.com/elyra-ai/airflow-notebook), which downloads the compressed input artifact archive from the cloud storage bucket, extracts the archive, processes the notebook or script, and uploads the output artifacts to the cloud storage bucket.
+
+1. Wait for the DAG run to finish.
+
+### Access the pipeline run outputs
+
+The DAG run outputs (completed notebooks, script output, and declared output files) are persisted in the cloud storage bucket you've configured in the runtime configuration.
+
+Elyra does not automatically download the output artifacts to your JupyterLab environment from the cloud storage bucket after DAG execution has completed. Therefore you have to use an S3 client, or, if configured, the cloud storage's web GUI to access the artifacts.
+
+1. Navigate to the bucket you've specified in the runtime configuration to review the content. 
+
+   ![Review run output artifacts](doc/images/inspect-object-storage-bucket-content.png)
+
+   The bucket contains for each node the following artifacts, which are prefixed with the DAG name:
+      - a `tar.gz` archive containing the notebook or script, and, if applicable, its declared input file dependencies
```

Comment:
Same as the Kubeflow tutorial where some commas around `for each node` may help readability